src/pages/workspaces/List.js

```diff
@@ -249,27 +251,16 @@ export const WorkspaceList = () => {
           },
           size: { basis: 120, grow: 1, shrink: 0 }
         }, {
-          headerRenderer: () => div({ className: 'sr-only' }, ['Actions']),
+          headerRenderer: () => div({ className: 'sr-only' }),
           cellRenderer: ({ rowIndex }) => {
-            const { accessLevel, workspace: { workspaceId, namespace, name }, ...workspace } = sortedWorkspaces[rowIndex]
+            const { accessLevel, ...workspace } = sortedWorkspaces[rowIndex]
             if (!Utils.canRead(accessLevel)) {
```

Comment:
This access level check was for the action menu, so I believe you should remove the check/short-circuit here so we can render the message about no workflows having been run instead of having a blank entry (see https://github.com/DataBiosphere/terra-ui/pull/3303/files#r942526064 for image).

I don't actually know if you will ever be able to see last run submission status if you don't have read access… I am assuming not, and for the workspaces where I don't have reader access `workspaceSubmissionStatus(workspace)` returns undefined, which the code handles as no submissions existing.